.travis.yml

```diff
@@ -19,6 +19,8 @@ after_success:
 - docker exec notes /edx/app/notes/notes/.travis/run_coverage.sh
 - codecov
 env:
+  - TOXENV=py38-django22
```

Comment:
Instead of adding a TOXENV here, could you add 3.8 to the python block above like this? https://github.com/edx/demographics/pull/29/files

Reply:
Hi @adzuci, the problem with that is that all the tests are run inside of the https://hub.docker.com/r/edxops/notes and it ignores the python version of the travis worker

Reply:
@adzuci I did that earlier but tests were running with py3.5 instead of py3.8.

Reply:
Ah, I see, if we install Python 3.8 in the dockerfile will that help, cc @tuchfarber

Reply:
@adzuci  the docker image used in the tests is created in the configuration repository and it already has [python3.8](https://github.com/edx/configuration/pull/5791).

The problem is that the default python in that image is python3.5

Reply:
Also note that pretty soon we'll want to add tests for Django 3.0 also, and that can only be accomplished via setting `TOXENV` appropriately and passing it through all necessary Docker and Makefile layers.